app/components/copy-webhook-url/copy-webhook-url.component.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
+
+angular.module('openshiftConsole')
+  .component('copyWebhookUrl', {
+    bindings: {
+      buildConfigName: "=",
+      triggerType: "=",
+      projectName: "=",
+      secret: "=",
+      webhookSecrets: "="
```

Comment:
For future reference, these should probably be one way bindings with `<` instead of `=`. But I don't want to hold the PR on that.